src/object/srv_mod.c

```diff
@@ -35,6 +37,7 @@ obj_mod_init(void)
 {
 	dss_abt_pool_choose_cb_register(DAOS_OBJ_MODULE,
 					ds_obj_abt_pool_choose_cb);
+	vos_dtx_register_check_leader(ds_pool_check_leader);
```

Comment:
after I read through ds_pool_check_leader() again, I think it probably should stay in object module?

Reply:
Then is it suitable for the DTX (dtx_resync) module to call the functions exported by object module?